src/archive.py

```diff
@@ -0,0 +1,21 @@
+"""Takes Issues Filed in the last week and puts them in a file for the upcoming stream"""
+import pathlib
+from datetime import datetime
+
+from src.engine import engine
+
+# Get the date of the next friday
```

Comment:
Is this comment left over from a previous version?

Reply:
yes